app/controllers/workflow_queues_controller.rb

```diff
@@ -50,7 +50,7 @@ def find_completed_objects
   end
 
   def workflows_for_step_and_scope(step, scope)
-    _repository, workflow, process = step.split(':')
+    workflow, process = step.split(':').last(2)
```

Comment:
This gives us more flexibility going forward.

If passed a step with a repository (*e.g.*, `dor:accessionWF:do-a-thing`), the repository is ignored and the workflow and process are set accordingly. In this way, the change is backwards-compatible with the former implementation. If, on the other hand, passed a step without a repository (*e.g.*, `accessionWF:do-a-thing`), this method will do the right thing: assign the workflow and step appropriately. This is an improvement over the past implementation and allows dor-workflow-client to begin phasing out the repository argument.

Reply:
is it worth it / stupid to notify honeybadger if the workflow doesn't end in 'WF' ? or some such?   Just in case?

Reply:
@ndushay I don't know that we've ever expressed a requirement that workflows must end in `'WF'`, so I'm meh on this change. If passed a bogus value for workflow or process, we'll get a HB alert eventually anyway :laughing: